config/initializers/sidekiq.rb

```diff
@@ -2,16 +2,17 @@
 require 'sidekiq-status'
 
 redis_url = "rediss://:#{ENV['REDIS_PASSWORD']}@#{ENV['REDIS_HOST']}:6379" if ENV['REDIS_HOST'].present? && ENV['REDIS_PASSWORD'].present?
+namespace = ENV.fetch('HOST', nil)
```

Comment:
Should the namespace default to something if not defined by `ENV['HOST']`? `'apply'` perhaps?

Reply:
Yes, you're right. So there is always a namespace and it is more predictable.